.github/workflows/tests.yml

```diff
@@ -43,29 +41,29 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v4
 
-      - name: Set up Python ${{ matrix.python-version }}
-        uses: actions/setup-python@v5
+      - name: Cache Docker images
+        uses: ScribeMD/docker-cache@fb28c93772363301b8d0a6072ce850224b73f74e  # 0.5.0
         with:
-          python-version: ${{ matrix.python-version }}
-          cache: pip
-          cache-dependency-path: Pipfile.lock
+          key: docker-${{ runner.os }}-${{ hashFiles('docker-services.yml') }}
```

Comment:
Not entirely correct to use the `docker-service.yml` file, since the pulled Docker images are the ones that come baked-in the fetched release of `docker-services-cli`... But it's good enough and we should probably keep the versions synced